docs/en/setup/backend/backend-mongodb-monitoring.md

```diff
@@ -0,0 +1,79 @@
+# MongoDB monitoring
```

Comment:
Please update doc menu for this as well.

Reply:
ok.